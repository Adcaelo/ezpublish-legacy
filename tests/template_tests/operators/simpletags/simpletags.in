{raw simpletags('Back To <strong>The</strong> Future')}
