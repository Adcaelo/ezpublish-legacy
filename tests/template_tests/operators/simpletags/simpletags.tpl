{'Back To <strong>The</strong> Future'|simpletags()}
