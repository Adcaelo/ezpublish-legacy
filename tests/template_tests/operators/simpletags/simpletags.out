Back To <b>The</b> Future
